applications/solvers/additiveFoam/additiveFoam.C

```diff
@@ -94,8 +94,19 @@ int main(int argc, char *argv[])
         }
 
         #include "thermo/TEqn.H"
-
-        ExaCA.update();
+       
+        // sensible cooling rate in liquid
+        volScalarField R
```

Comment:
Is there a reason you initialize this here, every timestep, instead of initializing once within `createFields.H` then recalculating before calling `ExaCA.update(R)`?

Reply:
Nope - I was primarily just testing out that interpolating the sensible cooling rate was correct. You are right, we should probably initialize this field and recalculate

Reply:
I went ahead and removed the change in calculating the cooling rate from this PR since it is not related to time stepping or speed up. Fixed in [be374d3](https://github.com/ORNL/AdditiveFOAM/pull/16/commits/be374d33f2660d38271ac4de8dfdf082505c545e)